craft_parts/executor/environment.py

```diff
@@ -140,6 +140,35 @@ def _basic_environment_for_part(part: Part, *, step_info: StepInfo) -> Dict[str,
     return part_environment
 
 
+def _get_step_environment(step_info: StepInfo) -> Dict[str, str]:
+    """Set environment variables from step information.
+
+    Variable names are prefixed by the application name and follows the
+    list of environment variables set by Snapcraft.
```

Comment:
```suggestion
    A list of environment variables prefixed by the application name.
```

Declare them if you want too :-)